### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by rubyimrie**

### CODE REVIEW

File: src/components/PieChart/style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.title{
```
Comment (RobinJehn) [inline src/components/PieChart/style.css]: Where is the pie chart used? Cannot find it

Comment (rubyimrie) [reply src/components/PieChart/style.css]: I've not used the pie chart component anywhere yet. I didn't mean to commit  the css for it that on this branch.

Comment (RobinJehn) [reply src/components/PieChart/style.css]: In that case, please remove it again and add it when you need it.

**Comment by RobinJehn** [comment]
Why did you delete the notification and do not disturb?

**Comment by RobinJehn** [comment]
Ah If I see it correctly, you now just moved the stuff into the other file?

File: src/components/PieChart/style.css (hunk 1 above)
Comment (RobinJehn) [reply src/components/PieChart/style.css]: This comment remains. Please delete the folder from this branch.

Comment (rubyimrie) [reply src/components/PieChart/style.css]: have done this now

**Pull request merged**